cr.h

```diff
@@ -436,6 +477,62 @@ static int cr_plugin_main(cr_plugin &ctx, cr_op operation);
 #define CR_STR "%ls"
 #define CR_INT "%ld"
 
+using so_handle = HMODULE;
+
+static std::wstring cr_utf8_to_wstring(const std::string &str) {
+    int wlen = MultiByteToWideChar(CP_UTF8, 0, str.c_str(), -1, 0, 0);
+    wchar_t wpath_small[MAX_PATH];
+    std::unique_ptr<wchar_t[]> wpath_big;
+    wchar_t* wpath = wpath_small;
+    if (wlen > _countof(wpath_small))
+    {
```

Comment:
Have you tried to run clang-format? I'm not sure if I've let it misconfigured or if it does not support it, I can't remember.
But { should be in the same line, also, any block even one-liners as in L493 should go inside braces, there are some if and a while without braces. So if you can fix it I'll be glad! :)

Reply:
Oh yeah i used clang-format, but did these changes after. Will fix it. Although clang-format does quite a lot of other changes that i found surprising arent already in the repository

Reply:
Yeah, I need check my clang-config and travis config anyway.
Just to note, you've missed some ifs and while without braces elsewhere. :P

Reply:
You mean ones with one line bodies? clang-format isnt adding braces to these. Should i?

Reply:
Yeah. I think this case is unsupported by clang-format. Thanks, I'll push his when I'll get at home.